app/home/components/mainSection/mainSection.scss

```diff
@@ -1,17 +1,27 @@
-$header-height: 75px;
+$header-height: 80px;
 $container-width: 1080px;
 .mainSectionContainer {
   padding-top: $header-height;
-  height: 800px;
+  height: 600px;
   overflow: hidden;
-  background-color: #f5f7fb;
+  background-image: url("https://s3.amazonaws.com/pluto-asset/background%402x.png");
```

Comment:
You can use https://assets.pluto.network/background%402x.png
and it's much faster than the current address.